build.sbt

```diff
@@ -47,10 +47,10 @@ lazy val microservice = Project(appName, file("."))
   .settings(scalaVersion := "2.12.10")
   .settings(
     majorVersion := 1,
-    //  addCompilerPlugin(scalafixSemanticdb),
+    addCompilerPlugin(scalafixSemanticdb),
     libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test
   )
-  .settings(scalacOptions ++= Seq("-Ypartial-unification", "-Yrangepos", "-Ywarn-unused:imports"))
+  .settings(scalacOptions ++= Seq("-Ypartial-unification", "-Yrangepos", "-Ywarn-unused:imports", "-P:semanticdb:exclude:^*.scala.html$|^*.routes$"))
```

Comment:
does this get rid of all the annoying warnings?

Reply:
This is suppose to exclude those files but it doesn't seem to work. I'm going to raise a bug with the ScalaMeta team.

Reply:
I'm not sure if it is the regular expression which specified incorrectly or if it is actually a bug.

Reply:
do you have to escape the dots in the regex?

Reply:
no, it's a bug in the semanticdb